frontend-react/src/components/DataDashboard/FacilitiesProviders/FacilitiesProvidersTable.tsx

```diff
@@ -16,37 +25,60 @@ const filterManagerDefaults: FilterManagerDefaults = {
     },
 };
 
-export default function FacilitiesProvidersTable() {
+interface ReceiverServicesProps {
+    receivers: RSReceiver[];
+    activeService: RSReceiver | undefined;
+    setActiveService: Dispatch<SetStateAction<RSReceiver | undefined>>;
+}
+
+function FacilitiesProvidersTableWithPagination({
+    receivers,
+    activeService,
+    setActiveService,
+}: ReceiverServicesProps) {
+    const featureEvent = `${FeatureName.FACILITIES_PROVIDERS} | ${EventName.TABLE_FILTER}`;
+
     const data = [
         {
+            aggregatorId: "12w3e4r",
```

Comment:
I know this is just test data, but all of the data you click have the same `12w3e4r` which means they all show that same `AFC Urgent Care` data. Would it be a lot of work to have the 3 sets of dummy data? Then we can have tests that click each link and show different rendered data.

Reply:
Its not the aggregatorId that determines the same `AFC Urgent Care` data, its the logic in the `linkBasePath` (ln 80) that determines what gets displayed. Considering this logic will be removed when we apply the new table I felt like it wasn't worth messing with.